src/main/java/io/orten/nano/model/Organization.java

```diff
@@ -1,37 +1,38 @@
 package io.orten.nano.model;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.Table;
+import org.hibernate.annotations.DynamicUpdate;
+import org.hibernate.annotations.SelectBeforeUpdate;
+
+import javax.persistence.*;
+import java.io.Serializable;
 /*
   represents an organization raising fund for one or more projects
  */
 
 @Entity
 @Table(name="organization")
-/*@SelectBeforeUpdate(value = true)
-@DynamicUpdate(value = true)*/
-public class Organization {
+@SelectBeforeUpdate(value = true)
+@DynamicUpdate(value = true)
+public class Organization implements Serializable{
 
     @Id
-    /*@GeneratedValue(strategy = GenerationType.IDENTITY)
-    public int id;*/
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    public int id;
     @Column(name="organizationID")
-    public String organizationID;
+    public String organizationId;
     @Column(name="name")
     public String name;
     @Column(name="address")
     public String address;
     @Column(name="contactPerson")
-    public String contactPerson;
-    @Column(name="contactPersonEmail")
+    public String contactPersonName;
+    @Column(name="contactEmail")
     public String contactPersonEmail;
     @Column(name="password")
     public String password;
     @Column(name="billingInformation")
     public String billingInformation;
-    @Column(name="accountNumber")
+    @Column(name="account_Number")
```

Comment:
Keep the naming convention uniform.
It is normal to keep all columns lowercase and a underscore between each word.
But most importantly: be consistent!